### PR TITLE
Simplify terraform ELB tasks

### PR DESCRIPTION
**Pull request by justinsb**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1181)

### CODE REVIEW

**Comment by justinsb** [comment]
@kris-nova mostly a reorg of the awstasks, to more closely map to the API.  Because terraform embeds the health check though, we also embed it - that differs from the API, but feels pretty natural.

File: upup/pkg/fi/cloudup/awstasks/load_balancer.go [hunk 1]
```diff
@@ -212,6 +213,12 @@ func (e *LoadBalancer) Find(c *fi.Context) (*LoadBalancer, error) {
 		actual.Listeners[loadBalancerPort] = actualListener
 	}
 
+	healthcheck, err := findHealthCheck(lb)
+	if err != nil {
+		return nil, err
```
Comment (chrislovecnm) [inline upup/pkg/fi/cloudup/awstasks/load_balancer.go]: Should we be fmt the error with a message?

**Comment by chrislovecnm** [comment]
Looks great, just a fmt that I would recommend

**Comment by kris-nova** [comment]
Can/should we close this now?

**Comment by justinsb** [comment]
This was part of the mega PR

**Pull request closed without merging**